src/player.js

```diff
@@ -0,0 +1,18 @@
+const player = {
+    name: '',
```

Comment:
If getters/setters are used we could use _varName notation to indicate the private variables.